prowler/lib/outputs/compliance/compliance.py

```diff
@@ -212,7 +212,7 @@ def get_check_compliance(finding, provider_type, output_options) -> dict:
                 compliance_fw = compliance.Framework
                 if compliance.Version:
                     compliance_fw = f"{compliance_fw}-{compliance.Version}"
-                if compliance.Provider == provider_type.upper():
+                if compliance.Provider.upper() == provider_type.upper():
```

Comment:
1. Please add tests for this function for other providers than AWS :D

Reply:
Done